docs/digital_analog_qc/analog-qubo.md

```diff
@@ -98,109 +92,102 @@ Q = np.array(
         [5.42015853, 0.85604541, 0.17675796, 0.32306662, -10.0],
     ]
 )
+
+# Cost function for a single bitstring
```

Comment:
Maybe add a formula ?

Reply:
No need, it is trivial and there is already ample reference for the reader to check it out by themselves.